extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
+lines = sensors_data.readlines()
+
+
+def sensors(lines):
+    #global lines
+    candidate_lines = []
+    for line in lines:
+        line = line.decode()
+        if line.startswith('kat.sensors'):
```

Comment:
No need to hardcode: `kat.sensors` - parse it as an argument. That way if someone wants to check for `kat.sensors.blabla` they won't need to manually change it here.

Reply:
@mamkhari

Reply:
@mamkhari what @mmphego is basically saying is that you need a variable for that `kat.sensors` and parse the variable into the function instead of the hard coded value 
`sensor_name = 'kat.sensors'` and parse `sensor_name` into the function